src/views/core/bodies/Single.vue

```diff
@@ -97,7 +97,7 @@
                     <span v-html="$options.filters.markdown(body.description)"/>
                   </td>
                 </tr>
-                <tr v-if="body.task_description !== null">
+                <tr v-if="body.task_description !== null && body.task_description != ''">
```

Comment:
Why both? Just to be sure?

Reply:
If the description was set, then deleted and saved, the field will no longer be null, but equal to an empty string. Hence the second part